index.html

```diff
@@ -3,25 +3,31 @@
 <head>
   <meta charset="utf-8">
   <title>Quantified Self - Diary</title>
+  <link rel="stylesheet" href="vendors/font-awesome-4.7.0/css/font-awesome.min.css">
   <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
 </head>
 <body>
 
-  <script src="main.bundle.js"></script>
+<script src="main.bundle.js"></script>
 <div class="container">
 
-<h1 class="text-center">Diary</h1><br>
+<h1 class="text-center">Diary </h1><br>
 <div class="row" id="meals">
 </div>
+<div class="row">
 
 <div class="col-md-6" id="totals-table">
-  <h3 class="text-center">Total</h3>
+  
 </div>
 
+<div class="col-md-6" id="foods-table">
```

Comment:
check your indentation in this file. be sure to indent properly to make it easier to read!